addons/captives/functions/fnc_doBlindfoldCaptive.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+ * Author: mrschick
+ * Puts a blindfold on a captive unit if the player has a blindfold in their inventory
+ *
+ * Arguments:
+ * 0: _unit-Player <OBJECT>
+ * 1: target <OBJECT>
+ *
+ * Return Value:
+ * The return value <BOOL>
+ *
+ * Example:
+ * [player, bob, true] call ACE_captives_fnc_doBlindfoldCaptive;
+ *
+ * Public: No
+ */
+
+params ["_unit", "_target", "_state"];
+
+// check if _unit has a Contact DLC blindfold in its inventory, abort otherwise.
+private _validBlindfolds = ["G_Blindfold_01_black_F", "G_Blindfold_01_white_F"];
+private _carriedBlindfoldIdx = _validBlindfolds findAny (items _unit);
+if (_carriedBlindfoldIdx == -1) exitWith {};
+
+// Remove _target goggles if it is wearing any and move them to _unit inventory
```

Comment:
What happens if the inventory is full? Should use the CBA function here.

Reply:
Good call. Implemented it by using [`CBA_fnc_canAddItem`](https://cbateam.github.io/CBA_A3/docs/files/common/fnc_canAddItem-sqf.html#CBA_fnc_canAddItem) and [`CBA_fnc_dropItem`](https://cbateam.github.io/CBA_A3/docs/files/common/fnc_dropItem-sqf.html#CBA_fnc_dropItem) in case both the captive and capturer's inventories are full.